data/na/swagger/1-0-5.yaml

```diff
@@ -1985,11 +2003,14 @@ definitions:
   CardResponse:
     description: 'For a 3D Secure payment process.'
     required:
-      - pa_res
+      - cres
     properties:
       pa_res:
         type: string
-        description: 'For a 3D Secure payment process this stores the Authentication Code (PaRes).'
+        description: 'For a 3D Secure payment process this stores the Authentication Code (PaRes).  NOTE: No longer used, this field is renamed to cres'.
```

Comment:
Is this true? I don't know for sure but I had assumed 3DSv1 would remain untouched, which means this value is still relevant for 3DSv1